content/en/docs/Kyverno CLI/_index.md

```diff
@@ -682,51 +682,23 @@ summary:
 
-Run tests on a set of local files:
+`test` works by scanning a given location, which can be either a Git repository or local folder, and executing the tests defined within. The rule types `validate` and `mutate` are currently supported. The command recursively looks for YAML files with policy test declarations (described below) with a specified file name and then executes those tests.  All files applicable to the same test must be co-located. Directory recursion is supported. `test` supports the [auto-gen feature](/docs/writing-policies/autogen/) making it possible to test, for example, Deployment resources against a Pod policy.
 
-```sh
-kyverno test /path/to/folderContainingTestYamls
-```
-
-#### Run tests on a Git repo:
-
-Testing on an entire repo by specifying branch name within repo URL:
-
-```sh
-kyverno test https://github.com/<ORG>/<REPO>/<BRANCH>
-```
-
-Example:
-
-```sh
-kyverno test https://github.com/kyverno/policies/release-1.5
-```
-
-{{% alert title="Note" color="info" %}}
-If `<BRANCH>` is not provided in this case, the default branch is assumed to be `main`.
-{{% /alert %}}
-
-Testing on a specific directory of the repo by specifying the directory within repo URL, and the branch with `--git-branch` flag _(shorthand `-b`)_:
-
-```sh
-kyverno test https://github.com/<ORG>/<REPO>/<DIR> --git-branch <BRANCH>
-```
+`test` will search for a file named `kyverno-test.yaml` and, if found, will execute the tests within.
 
-Example:
+In each test, there are three desired results which can be tested for. If the actual result of the test, once executed, matches the desired result as defined in the test manifest, it will be scored as a `pass` in the command output. For example, if the specified result of a given test of a resource against a policy is declared to be a `pass` and the actual result when tested is also a `pass`, the command output will show as `pass`. If the actual result was instead a `skip`, the command output will show as `fail` because the two results do not agree. The following are the desired results which can be specified in a test manifest.
 
-```sh
-kyverno test https://github.com/kyverno/policies/pod-security/restricted --git-branch release-1.5
-```
+1. pass: The resource passes the policy definition. For `validate` rules which are written with a `deny` statement, this will not be a possible result. `mutate` rules can declare a pass.
+2. skip: The resource does not meet either the `match` or `exclude` block, or does not pass the `preconditions` statements. For `validate` rules which are written with a `deny` statement, this is a possible result. If a rule contains certain conditional anchors which are not satisfied, the result may also be scored as a `skip`.
+3. fail: The resource does not pass the policy definition. Typically used for `validate` rules with pattern-style policy definitions.
```

Comment:
It is also possible to add a `warn` state. See https://github.com/kyverno/kyverno/pull/3821

Reply:
That's for apply. Does it work for test?

Reply:
Yup, there is a linked test case for `test` as well.

Reply:
See https://github.com/moolen/kyverno/blob/f83c398bad078b519f735121cb7b17d11ffad75d/test/cli/test/depecated_apis/kyverno-test.yaml#L16

Reply:
Thank you for catching that! This is why I love peer review. Corrected in https://github.com/kyverno/website/pull/556/commits/5fc5ef71aaf27914e8475d4a42a28d901b69df07